cob_obstacle_distance/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cob_obstacle_distance)
+
+set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
+
+# Load catkin and all dependencies required for this package
+find_package(catkin REQUIRED COMPONENTS cmake_modules roscpp sensor_msgs geometry_msgs nav_msgs tf tf_conversions dynamic_reconfigure message_generation kdl_parser kdl_conversions)
+
```

Comment:
please verify package.xml and CMakeList.txt 
`nav_msgs` oboslete
`visualization_msgs` missing